content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst

```diff
@@ -0,0 +1,108 @@
+.. _plm/eco/eco-type:
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+========
+ECO type
+========
+
+Organize engineering change orders (ECOs) into a *ECO type* project and create custom stages to
+track the progress of changes within a structured Gantt view project pipeline. This categorization
+ensures that collaborators and stakeholders only view and assist with relevant |BOM| improvements.
+
+Create ECO type
+===============
+
+To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
+Types`.
+
+Create a new ECO type by clicking :guilabel:`New`. On the new :guilabel:`ECO Types` form, fill in
+the following information:
+
+- :guilabel:`Name`: the name of the |ECO| and will organize all of the |ECOs| of this *type* in a
+  project.
+- :guilabel:`Email Alias`: an optional field. If filled, emails submitted to this email address will
+  automatically generate |ECOs| in the left-most stage of this |ECO| type.
+
+.. example::
+   The `Formulation change` |ECO| type is used to organize and track related |ECOs| in a single
+   project. Configuring the :guilabel:`Email Alias` field generates |ECOs| in the `Formulation
+   change` project sent to the email address, `pawlish-change@pawlished-glam.odoo.com`.
+
+   .. image:: eco_type/create-eco-type.png
+      :align: center
+      :alt: Example of an ECO type.
+
+Edit ECO type
+=============
+
+Modify existing |ECO| type names and email aliases by clicking by navigating to the
+:menuselection:`PLM app --> Configuration --> ECO Types` page. There, click on the desired on the
+desired |ECO| type from the list. On the form for each |ECO| type, edit the :guilabel:`Name` and
+:guilabel:`Email Alias` fields.
+
+.. _plm/eco/stage-config:
+
+Stage configuration
+===================
+
+*Stages* in each |ECO| type help to organize and track the progress of |ECOs|. Additionally,
+approvers can be configured on each stage to enforce reviews on changes and prevent errors on the
```

Comment:
Are they really enforcing reviews or are they approving suggested changes? I think the phrasing here could be reworked.